README.md

```diff
@@ -369,15 +369,15 @@ DomainExtractor.parse_query_params(query_string)
 
 ```ruby
 urls = scrape_page_links(page)
-domains = urls.map { |url| DomainExtractor.parse(url)&.dig(:root_domain) }.compact.uniq
+domains = urls.map { |url| DomainExtractor.parse(url).root_domain }.compact.uniq
 ```
 
 **Analytics & Tracking**
 
 ```ruby
 referrer = request.referrer
 parsed = DomainExtractor.parse(referrer)
-track_event('page_view', source_domain: parsed[:root_domain]) if parsed
+track_event('page_view', source_domain: parsed[:root_domain])
```

Comment:
Here `parsed` will always be truthy since it is an instance of `DomainExtractor::ParsedURL`.